src/network_interfaces.jl

```diff
@@ -0,0 +1,136 @@
+## This is a minimal version of NetworkInterfaceControllers.jl
```

Comment:
There should be a notice that the license is MIT, since it's a derivative work.

Reply:
Good point, added in f4e0443.